modules/bemenu.nix

```
@@ -0,0 +1,34 @@
+{pkgs, config, lib, ... }:
+
+with config.lib.stylix.colors.withHashtag;
+with config.stylix.fonts;
+{
+  options.stylix.targets.bemenu = {
+    enable = config.lib.stylix.mkEnableTarget "bemenu" true;
+    fontSize = lib.mkOption { type = lib.types.str; default = ""; }; # optional argument
```

This should probably be `int` type:

```suggestion
    fontSize = lib.mkOption {
      type = with lib.types; nullOr int;
      default = null;
    };
```

Then convert it to the appropriate string where it's needed:

```nix
${lib.optionalString (fontSize != null) (builtins.toString fontSize)}
```

It's a bit more wordy, but it catches potential mistakes like setting `"bold"` as the font size.